pkg/controller/nodes/task/resourcemanager/resourcemanager.go

```diff
@@ -2,14 +2,37 @@ package resourcemanager
 
 import (
 	"context"
+	"fmt"
 	"sync"
 
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+
 	pluginCore "github.com/lyft/flyteplugins/go/tasks/pluginmachinery/core"
 	"github.com/lyft/flytestdlib/promutils"
 )
 
 //go:generate mockery -name ResourceManager -case=underscore
 
+type TokenNamespace string
```

Comment:
Do we need a different type for Token vs Resource Namespaces? are they really different?